Python/Product/PythonTools/PythonTools/Options/OptionsTreeView.cs

```diff
@@ -42,28 +39,16 @@ class OptionsTreeView : TreeView {
 
         private void InitializeImageList() {
             if (_imageList == null) {
-                using (var imgHandler = new ImageHandler(typeof(OptionsTreeView).Assembly.GetManifestResourceStream("Microsoft.PythonTools.Project.Resources.imagelis.bmp"))) {
-#pragma warning disable 0618
-                    var openFolder = imgHandler.ImageList.Images[(int)ProjectNode.ImageName.OpenFolder];
-                    var closedFolder = imgHandler.ImageList.Images[(int)ProjectNode.ImageName.Folder];
-#pragma warning restore 0618
-
-                    _imageList = new ImageList();
-
-                    Bitmap bmp = new Bitmap(16, 16);
-
-                    // transparent image is the image we use for owner drawn icons
-                    _imageList.TransparentColor = Color.Magenta;
-                    using (var g = Graphics.FromImage(bmp)) {
-                        g.FillRectangle(
-                            Brushes.Magenta,
-                            new Rectangle(0, 0, 16, 16)
-                        );
-                    }
-                    _imageList.Images.Add(bmp);
-                    _imageList.Images.Add(openFolder);
-                    _imageList.Images.Add(closedFolder);
+                _imageList = new ImageList();

```

Comment:
Is it a problem if StateImageList is initialized to null?  If not, we could remove _imageList completely.